src/m_refraction.cc

```diff
@@ -390,6 +432,29 @@ void complex_refr_indexIceMatzler06(GriddedField3& complex_refr_index,
   }
 }
 
+/* Workspace method: Doxygen documentation will be auto-generated */
+void complex_refr_indexTConstant(GriddedField3& complex_refr_index,
```

Comment:
The temperature seems to not be used locally by this function so it should probably be dropped from the name